test/fixtures/PluginTestProvider.ts

```diff
@@ -0,0 +1,50 @@
+/* Copyright © 2021 Seneca Project Contributors, MIT License. */
+
+
+function PluginTestProvider(this: any, _options: any) {
```

Comment:
remove all uses of the term "provider", as this is not a provider

Reply:
Old